compressor/tests/test_offline.py

```diff
@@ -364,6 +357,18 @@ class OfflineCompressStaticTemplateTagTestCase(OfflineTestCaseMixin, TestCase):
     templates_dir = 'test_static_templatetag'
     expected_hash = 'dfa2bb387fa8'
 
+    def _test_offline(self, engine):
+        count, result = CompressCommand().compress(
+            log=self.log, verbosity=self.verbosity, engine=engine
+        )
+        self.assertEqual(1, count)
+        self.assertEqual([self._render_script(self.expected_hash)], result)
+        self.assertEqual(self._render_template(engine), self._render_result(result))
+
+        # Changing settings.STATIC_URL doesn't affect the next "offline" decompression
+        with self.settings(STATIC_URL='/another/static/url/'):
+            self.assertEqual(self._render_template(engine), self._render_result(result))
```

Comment:
could you make it more explicit what is being tested here? i don't fully understand the structure of the tests but my feeling is this should go in a new Class whose name roughly describes what's being tested. 

also, could you clarify what you mean by "decompression" in the comment? we currently don't use that word anywhere